bower.json

```diff
@@ -23,7 +23,7 @@
     "cf-icons": "^1.0.0",
     "cf-pagination": "^1.0.0",
     "cf-typography": "^1.0.0",
-    "cf-layout": "^1.0.0"
+    "cf-layout": "^2.0.0"
```

Comment:
The last discussion we had in the FEWD call was that individual modules would keep their own versions and only CF itself would update whenever one of the modules did. I think that's the smartest route because someone installing just cf-core, shouldn't need to update their deps when we release the next patch to cf-core.